.travis.yml

```diff
@@ -345,7 +470,7 @@ install:
   - git submodule init libs/xpressive
 
   - git submodule init libs/headers tools/boost_install tools/build
-  - git submodule update
+  - git submodule update --jobs `(nproc || sysctl -n hw.ncpu)`
```

Comment:
You can set this to something like 10 because it is not CPU bound, but network IO latency bound.